pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -588,7 +588,7 @@ protected void setupCustomRouting(@NonNull Config cfg) {
         get(apiRoot + RoutePath.CLINICAL_KIT_ENDPOINT, new ClinicalKitsRoute(notificationUtil), new JsonTransformer());
 
         if (!cfg.getBoolean("ui.production")) {
-            NonPepperKitCreationService nonPepperKitCreationService = new NonPepperKitCreationService();
+            NonPepperKitCreationService nonPepperKitCreationService = new NonPepperKitCreationService(new NonPepperStatusKitService());
```

Comment:
May be have NonPepperKitCreationService instantiate this NonPepperStatusKitService !

Reply:
Agree. This would make sense if you were planning to pass a mock, but I don't see that. Also, see my comment below.

Reply:
Agreed and thanks for catching this! Fixed now